specs/state-proof.md

```diff
@@ -92,25 +98,33 @@ to not be in the table.
 
 ### Tx Access List Account
 - 9.0. `field_tag` and `storage_key` are 0
-- 9.1. `state root` is the same
-- 9.2. First access for a set of all keys are 0 if `READ`
-
+- 9.1. `value` is boolean
+- 9.2. `initial_value` is false
+- 9.3. `state root` is the same
+- 9.4. First access for a set of all keys are 0 if `READ`
+- 9.5. `value` column at previous rotation equals `value_prev` at current rotation
 
 ### Tx Access List Account Storage
 - 10.0. `field_tag` is 0
-- 10.1. `state root` is the same
-- 10.2. First access for a set of all keys are 0 if `READ`
+- 10.1. `value` is boolean
+- 10.2. `initial_value` is false
+- 10.3. `state root` is the same
+- 10.4. First access for a set of all keys are 0 if `READ`
+- 10.5. `value` column at previous rotation equals `value_prev` at current rotation
 
 ### Tx Log
 - 11.0. `is_write` is 1
```

Comment:
In the code, it is written "is true". Should we go for "is 1" or "is true"?

Reply:
Fixed in [11.0 is_write -> true](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/520/commits/8a2f518a07557f42b23b3eac29cb48369c4c78bf)